src/vending-machine.js

```diff
@@ -0,0 +1,61 @@
+const processData = require('./../__mock__/process-data');
+
+class vendingMachine {
+    constructor(data) {
+        this.data = processData(data);
```

Comment:
Couple issues here:

- The processData function doesn't do anything, so the call is unnecessary
- You should never include mock files in actual files. Files in `__mocks__` are only for testing